pyproject.toml

```
@@ -18,7 +18,7 @@ include = [ "alpaca/py.typed" ]
 python = "^3.8.0"
 requests = "^2.30.0"
 pydantic = "^2.0.3"
-pandas = "^2.0.0"
+pandas = ">=1.5.3"
```

how about adding upper limit for safe?
```suggestion
pandas = ">=1.5.3, <3.0.0"
```

---

after considering this. please let me change my mind. We can not know the future. So, the upper band should be managed by users.